### PR TITLE
New seeds deployed + vesting logic

### PR DESCRIPTION
**Pull request by codebuster22**

### CODE REVIEW

**Comment by codebuster22** [comment]
Don't merge now, I'll deploy contracts again, just to be sure they are not deployed twice. Till now have managed to resolve merge conflict.

File: contracts/seed/Seed.sol [hunk 1]
```diff
@@ -36,7 +36,7 @@ contract Seed {
     uint256 public feeAmountRequired;     // Amount of seed required for fee
     uint256 public price;
     uint256 public startTime;
-    uint256 public endTime;
+    uint256 public endTime;               // end time set by project admin, last restort end time, if max not reached
```
Comment (dkent600) [inline contracts/seed/Seed.sol:39]: "// set by project admin, this is the last resort endTime to be applied when maximumReached has not been reached by then"

File: contracts/seed/Seed.sol [hunk 2]
```diff
@@ -55,6 +55,8 @@ contract Seed {
     bool    public initialized;            // is this contract initialized [not necessary that it is funded]
     bool    public minimumReached;         // if the softCap[minimum limit of funding token] is reached
     bool    public maximumReached;         // if the hardCap[maximum limit of funding token] is reached
+    uint256 public vestingStartTime;       // timestamp when vesting start, by default = endTime,
+                                           //     correct time only when distribution ends
```
Comment (dkent600) [inline contracts/seed/Seed.sol:59]: i don't think this second line is needed. doesn't make sense to me anyway

File: contracts/seed/Seed.sol [hunk 3]
```diff
@@ -55,6 +55,8 @@ contract Seed {
     bool    public initialized;            // is this contract initialized [not necessary that it is funded]
     bool    public minimumReached;         // if the softCap[minimum limit of funding token] is reached
     bool    public maximumReached;         // if the hardCap[maximum limit of funding token] is reached
+    uint256 public vestingStartTime;       // timestamp when vesting start, by default = endTime,
```
Comment (dkent600) [inline contracts/seed/Seed.sol:58]: "// timestamp for when vesting starts, by default == endTime, otherwise when maximumReached is reached"

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: contracts/seed/Seed.sol (hunk 1 above)
Comment (codebuster22) [reply contracts/seed/Seed.sol:39]: I'll have to add in two lines, because of lint rules.

**Review by dkent600 — APPROVED** [review]

**Pull request merged**